tests/bugs/gh-3637.slang.3.expected.txt

```diff
@@ -0,0 +1,4 @@
+type: uint64_t
```

Comment:
With the `filecheck` directive, you can delete this `.expected.txt` file.

Reply:
Sure, will do.
(I just jumped back from #3825), and will also take care of with the one test failure.

Reply:
File is removed.